app/models/contact.rb

```diff
@@ -12,7 +12,8 @@ def headers
     {
       subject: "#{subject} - Contato do Site",
       to: "contato@meucontroledeponto.com.br",
-      from: %("#{name}" <#{email}>)
+      from: "info@meucontroledeponto.com.br",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.